Mainframe3270/x3270.py

```diff
@@ -106,6 +112,21 @@ def open_connection(
         else:
             self.mf.connect(f"{host_string}:{port}")
 
+        self.height = height
+        self.width = width
+
+        # Explictly configuring model based on width and height
```

Comment:
Please avoid comments in the code. If it is not self explanatory, implement it in a way that makes it so